multi-node/aws/pkg/config/config.go

```diff
@@ -151,7 +153,7 @@ func (c Cluster) Config() (*Config, error) {
 	config.APIServers = fmt.Sprintf("http://%s:8080", c.ControllerIP)
 	config.SecureAPIServers = fmt.Sprintf("https://%s:443", c.ControllerIP)
 	config.APIServerEndpoint = fmt.Sprintf("https://%s", c.ExternalDNSName)
-	if config.UseCalico {
+	if config.UseCalico || config.ContainerRuntime == "rkt" {
```

Comment:
Added a commit that moves everything to cni and takes away the complication here.